From 73f5788b510d7934b6094d679945c7901b982c4f Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Mon, 4 Jan 2016 13:20:51 +0100
Subject: [PATCH 20/29] ARM: socfpga: dts: Add altera specific compatible to
 sdr regs

The SDRAM control regs must also have alt,sdr-ctl property, otherwise the
FPGA bridge driver cannot locate it through the syscon interface.

Signed-off-by: Marek Vasut <marex@denx.de>
---
 arch/arm/boot/dts/socfpga.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index bd50303..7f4ef1e 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -703,7 +703,7 @@
 		};
 
 		sdr: sdr@ffc25000 {
-			compatible = "syscon";
+			compatible = "altr,sdr-ctl", "syscon";
 			reg = <0xffc25000 0x1000>;
 		};
 
-- 
2.10.2

